py-polars/polars/expr/expr.py

```diff
@@ -3372,17 +3372,15 @@ def rolling(
             {UInt32, UInt64, Int32, Int64}. Note that the first three get temporarily
             cast to Int64, so if performance matters use an Int64 column.
         period
-            length of the window - must be non-negative
+            Length of the window - must be non-negative.
         offset
-            offset of the window. Default is -period
+            Offset of the window. Default is `-period`.
         closed : {'right', 'left', 'both', 'none'}
             Define which sides of the temporal interval are closed (inclusive).
         check_sorted
-            When the `by` argument is given, polars can not check sortedness
-            by the metadata and has to do a full scan on the index column to
-            verify data is sorted. This is expensive. If you are sure the
-            data within the by groups is sorted, you can set this to `False`.
-            Doing so incorrectly will lead to incorrect output
+            Whether to check that `index_column` is sorted.
+            If you are sure the data is sorted, you can set this to `False`.
+            Doing so incorrectly will lead to incorrect output.
```

Comment:
`Expr.rolling` doesn't have a `by` argument, so I've removed it from the description